discovery-infra/tests/test_wrong_boot_order.py

```diff
@@ -20,12 +25,165 @@ def test_wrong_boot_order_one_node(self, nodes, cluster):
 
         # Wait until wrong boot order
         new_cluster.wait_for_one_host_to_be_in_wrong_boot_order()
+        new_cluster.wait_for_cluster_to_be_in_installing_pending_user_action_status()
 
-        # Reboot required nodes into ISO
+        # Reboot required nodes into HD
         node.shutdown()
         node.set_boot_order(cd_first=False)
         node.start()
 
         # wait until all nodes are in Installed status, will fail in case one host in error
+        new_cluster.wait_for_cluster_to_be_in_installing_status()
+        new_cluster.wait_for_hosts_to_install()
+        new_cluster.wait_for_install()
+
+    @pytest.mark.regression
```

Comment:
Im a bit concern about it, cause this tests wont run on Staging (not yet in)
please remove the regressions mark for now. instead we can have another mark representing new_feature \ master?
@nshidlin @RazRegev @tsorya WDYT?

Reply:
I think just leaving the regression mark off is good. Then test runs against staging will just run regression mark

Reply:
OK sounds good

Reply:
not completely actually, in this case the wrong boot order wont run at all (cause also the existing one changes)
we must have branches in some point

Reply:
@RazRegev please leave the existing wrong boot order tests as is (as master) with regression mark (for staging running).
for all new tests (4 in total), do not add any mark.